docs/pressable.md

```diff
@@ -0,0 +1,184 @@
+---
+id: pressable
+title: Pressable
+---
+
+Pressable is a Core Component wrapper that can detect various types of interaction states on any of it's defined children.
+
+```jsx
+<Pressable onPress={onPressFunction}>
+  <Text>I'm pressable!</Text>
+</Pressable>
+```
+
+## Example
+
+```
+import React, { useState } from 'react';
+import {
+  Pressable,
+  StyleSheet,
+  Text,
+  View,
+} from 'react-native';
+
+export default App = () => {
+  const [timesPressed, setTimesPressed] = useState(0);
+
+  let textLog = '';
+  if (timesPressed > 1) {
+    textLog = timesPressed + 'x onPress';
+  } else if (timesPressed > 0) {
+    textLog = 'onPress';
+  }
+
+  return (
+    <View>
+      <Pressable
+          onPress={() => {
+            setTimesPressed(current => current + 1);
+          }}>
+          {({pressed}) => (
+            <Text
+              style={[styles.text, pressed ? styles.textPress : styles.textRest]}>

```

Comment:
This isn't quite what I meant, I meant using the `style` prop on Pressable, so that it shows that it takes a function just like `children` do. 

I was thinking of having it use style like this example from RNTester: https://github.com/facebook/react-native/blob/2c473e1a38c35957fe80b6c334a1983c034c2bbc/RNTester/js/examples/Pressable/PressableExample.js#L333-L336